modules/sdk-coin-sol/src/sol.ts

```diff
@@ -562,8 +562,8 @@ export class Sol extends BaseCoin {
    */
   async recover(params: RecoveryOptions): Promise<SolTx> {
     const isUnsignedSweep = !params.userKey && !params.backupKey && !params.walletPassphrase;
-    const startingDerivationIndex = params.startingScanningIndex ? params.startingScanningIndex : 0;
-    const endingDerivationIndex = params.scan ? params.scan : 4294967295; // largest positive 32 bit int
```

Comment:
using lodash to check for undefined now, because when the `starting` and `ending` indices are 0, they are actually falsey values